papermill/cli.py

```diff
@@ -1,38 +1,66 @@
+# -*- coding: utf-8 -*-
+
+"""Main `papermill` interface."""
+
 import base64
+
 import click
 import yaml
+
 from papermill.execute import execute_notebook
 from papermill.iorw import read_yaml_file
 
 
-@click.command()
-@click.argument('notebook')
-@click.argument('output')
-@click.option('--parameters', '-p', help="Parameters to pass to the parameters cell.", multiple=True, nargs=2)
-@click.option('--raw_parameters', '-r', help="Parameters to be read as raw string.", multiple=True, nargs=2)
-@click.option('--parameters_file', '-f', help="Path to YAML file containing parameters.")
-@click.option('--parameters_yaml', '-y', help="YAML string to be used as parameters.")
-@click.option('--parameters_base64', '-b', help="Base64 encoded YAML string as parameters.")
-@click.option('--kernel', '-k', help="Kernel name to run the notebook with.")
-def papermill(notebook, output, parameters, raw_parameters, parameters_file, parameters_yaml, parameters_base64,
-              kernel):
+@click.command(context_settings=dict(help_option_names=['-h', '--help']))
+@click.argument('src')
+@click.argument('dst')
+@click.option(
+    '--param', '-p',
+    help='Parameters to pass to the parameters cell.',
+    multiple=True, nargs=2
+)
+@click.option(
+    '--raw_param', '-r',
+    help='Parameters to be read as raw string.',
+    multiple=True, nargs=2
+)
+@click.option(
+    '--file_param', '-f',
```

Comment:
What would you think of `--params_via_file` or `--param_file`? It's really hard for me to read `file_param` and not think of it as "the file param" which I would interpret as being a redundant version of a "file" param.